Jenkinsfile

```diff
@@ -0,0 +1,117 @@
+pipeline {
+    agent {
+        label 'linux && python'
+    }
+    stages {
+        stage('Build') {
+            steps {
+                echo 'Building..'
+                sh '''
+                    virtualenv ./sdk-deploy
+                    . ./sdk-deploy/bin/activate
+                    python setup.py sdist
+                    python setup.py bdist_wheel
+                '''
+            }
+        }
+        stage('Test') {
+            steps {
+                echo 'Testing..'
+                sh '''
+                    . ./sdk-deploy/bin/activate
+                    python setup.py test
```

Comment:
if not using tox for multiple version testing, also consider using the makefile.